target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -162,6 +162,11 @@ dlink,dwr-116-a1|\
 mzk-ex300np)
 	set_wifi_led "$boardname:green:wifi"
 	;;
+dlink,dwr-921-c1)
+	set_wifi_led "$boardname:green:wifi"
+	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "br-lan"
```

Comment:
You really make it harder than it has to be...

Please don't do it this way. You most likely will not find any example in the tree using a not physical interface with the netdev trigger.

Please use `ucidef_set_led_switch` as  suggested. The hex value used for the trigger is binary mask selecting the switch ports which should trigger the LED. It has to be read from right (port 0) to left (port 4). For your board the following should do the trick:

```
ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x0f"
```

Reply:
done.